pkg/app/piped/eventwatcher/eventwatcher.go

```diff
@@ -199,21 +217,21 @@ func (w *watcher) updateValues(ctx context.Context, repo git.Repo, events []conf
 
 	w.logger.Info(fmt.Sprintf("event watcher will update %d outdated values", len(commits)))
 	for _, c := range commits {
-		if err := tmpRepo.CommitChanges(ctx, tmpRepo.GetClonedBranch(), c.message, false, c.changes); err != nil {
+		if err := repo.CommitChanges(ctx, repo.GetClonedBranch(), c.message, false, c.changes); err != nil {
```

Comment:
The root cause is that it tries to edit files before committing even though we've already written to files in L254.

So I settled on using `tmpRepo` only for local editing.